packages/angular/projects/ui-angular/src/lib/services/authenticator.service.ts

```diff
@@ -24,21 +23,23 @@ const logger = new Logger('state-machine');
 export class AuthenticatorService implements OnDestroy {
   private _authState: AuthMachineState;
   private _authService: AuthInterpreter;
-  private _sendEventAliases: ReturnType<typeof getSendEventAliases>;
   private _machineSubscription: Subscription;
-  private _facade: ReturnType<typeof getServiceContextFacade>;
+  private _facade: ReturnType<typeof getServiceFacade>;
```

Comment:
While I was here, I made a small refactor to store facade values in one variable `_facade` instead of two variables.

For context, `AuthenticatorService` maintained two variables:
- `_facade`, which contains context facade values (e.g. `route`)
- `_sendEventAliases`, which contains transition methods like `toSignIn()`

Combining these two allows for performance boosts, because we can now do:

```diff
- return { ..._facade, ..._sendEventAliases };
+ return _facade;
```
to avoid unnecessary destructures.

Reply:
👍